RELEASING.md

```diff
@@ -24,7 +24,7 @@ You will also need the following credentials:
 Easy copy for editing all the version files:
 
 ```bash
-$EDITOR compute_sdk/globus_compute_sdk/version.py compute_endpoint/setup.py compute_endpoint/globus_compute_endpoint/version.py compute_funcx/sdk/setup.py compute_funcx/endpoint/setup.py compute_funcx/sdk/funcx/version.py compute_funcx/endpoint/funcx_endpoint/version.py
+$EDITOR compute_sdk/globus_compute_sdk/version.py compute_endpoint/setup.py compute_endpoint/globus_compute_endpoint/version.py
```

Comment:
Do we also remove the `compute_funcx/` directory at this time?

Reply:
We could, but there is a non-zero chance we will release another update to _funcx?  If we do remove the directory, that should be its own PR (remove deprecated funcx) instead of a release bump one.

Reply:
That's fair.